.travis.yml

```diff
@@ -7,22 +7,17 @@ notifications:
 node_js:
 - '8'
 before_install:
-- npm install -g npm@5
-- npm install -g greenkeeper-lockfile@1
-jobs:
-  include:
-  - stage: test
-    script:
-    - npm run lint
-    - npm run test
-    - npm run compile-prod
-    - npm run is-es5
-    before_script: greenkeeper-lockfile-update
-    after_script: greenkeeper-lockfile-upload
-  - stage: gh-pages
-    if: branch = master
-    script:
-    - npm run deploy-storybook-ci
```

Comment:
So this removes greenkeeper and storybook?

Reply:
I intended to remove greenkeeper since we think we want to transition to renovate, but I wasn't sure about story book, can I add that npm run line to this block or does it need to be a separate stage (still figuring our stages: https://docs.travis-ci.com/user/build-stages)

This is the repo I'm basing the changes off of: https://github.com/edx/frontend-base
& the slack thread about `edx-semantec-release` Github flow: https://edx-internal.slack.com/archives/CCE8D250D/p1568648293009400

Reply:
I think it's fine moving to renovate over greenkeeper, but storybook isn't being replaced with an alternative for developing locally so I wouldn't want to see it removed.

Reply:
Cool, I added that script back in and force pushed, could you describe how we use storybook? https://www.npmjs.com/package/@storybook/react